.travis.yml

```diff
@@ -23,7 +23,23 @@ matrix:
           packages: ['g++-6']
       env: COMPILER='g++-6'
 
-#    # 3/ OSX Clang Builds - disabled for now
+    # 3/ Linux GCC 4.9 Builds
+    - os: linux
+      compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-4.9
+            - g++-6
```

Comment:
Why version 6 also?

Reply:
Update alternatives is called for all Linux builds to select gcc-6.0. It needs to exist.

It is not necessarily selected as the C++ compiler.

There may be a better way to do this.

Reply:
It's what I had to do in my own travis to get 4.9 to install correctly